packages/jalaali/src/jalaali-utils.ts

```diff
@@ -25,6 +25,7 @@ type Moment = jMoment.Moment;
 const defaultFormats: DateIOFormats = {
   dayOfMonth: "jD",
   fullDate: "jYYYY, jMMMM Do",
+  fullDateWithWeekday: "jYYYY, jMMMM Do, dddd",
```

Comment:
```suggestion
  fullDateWithWeekday: "dddd Do jMMMM jYYYY",
```

Also I believe `fullDate` and `fullDateTime` patterns are not correct.